source/elements/oneDPL/source/sycl_kernels_api.rst

```diff
@@ -0,0 +1,42 @@
+.. SPDX-FileCopyrightText: 2019-2022 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+SYCL Kernels API
+----------------
+
+Supported C++ Standard Library APIs and Algorithms
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+oneDPL defines a subset of the C++ standard library APIs for use in DPC++ kernels. These APIs
+can be employed in the kernels similarly to how they are employed in code for a typical
+CPU-based platform.
+
+For all C++ algorithms accepting execution policies (as defined by `C++ Standard`_), oneDPL provides
+an implementation for oneAPI devices via :code:`oneapi::dpl::execution::device_policy`. These algorithms
+must be capable of processing data in SYCL buffers (passed via :code:`oneapi::dpl::begin/end`)
+and in unified shared memory (USM). (See :doc:`Parallel API <parallel_api>`.)
```

Comment:
Let's move this block to the beginning of the Parallel API doc.

Reply:
Applied suggestion